knowledge-portal/source/README.md

```diff
@@ -1,61 +1,67 @@
-# Usage
+# Knowlege portal Source Pipeline
 
-## Adding this pipeline to your repo
+## Usage
+
+### Adding this pipeline to your repo
 
 You will have to modify the existing `gitlab-ci.yml` file to include:
 
 ```yaml
-    stages:
-        - push
-    include:
-        - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+stages:
+  - push 
+
+include:
+  - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+
+variables:
+  GIT_STRATEGY: clone
 
 ```
-## Configure CI/CD variables (applied on overall gitlab instance)
+### Using the toml file to connect to Knowledge Portal
```

Comment:
*to connect your source git repository to the knowledge portal

Reply:
done, instead used "Connect your Source Git repository to the Knowledge Portal"